docs/inc-examples.md

```diff
@@ -1,8 +1,8 @@
 Every module must contain at least one example.
 
 - An example is a [root terraform module](https://www.terraform.io/language/modules#the-root-module) that uses the module and demonstrates its usage.
-- Examples are located in the `/examples` directory. See [module structure](module-structure.md).
-- After you add an example, link to it from the Examples section in your root-level readme file. See the [readme](https://github.com/terraform-ibm-modules/terraform-ibm-module-template#examples) file from the module template.
+- Examples are located in the `examples` directory. See [module structure](module-structure.md).
+- The main readme file includes links to your examples through a pre-commit hook that creates the links when a pull request passes the CI pipeline.
```

Comment:
Not quite right. The pre-commit hooks need to run locally first in order to add the examples to the readme. If you didn't run them locally and just committed the code, the hook would fail in the PR pipeline, and the pipeline would be marked as failed.
the way it is currently phrased it sounds like the examples would automatically be added in the PR, which is only the case for PRs auto generated by renovate

Reply:
OK. I added a step to run the pre-commit check in a couple of places because we didn't document that before they create the PR. Also updated the text about examples that shows up in a couple of places. 

You can see the output as it will publish:
- https://pages.github.ibm.com/allen-dean/tim-docs/#/contribute-module?id=create-or-update-examples
- https://pages.github.ibm.com/allen-dean/tim-docs/#/migrate-module?id=create-or-update-examples